Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -965,6 +966,11 @@ bool RenderLayerCompositor::updateCompositingLayers(CompositingUpdateType update
         LOG(Compositing, "%8d %11d %9d %20.2f %22.2f %22.2f %18.2f\n",
             m_obligateCompositedLayerCount + m_secondaryCompositedLayerCount, m_obligateCompositedLayerCount,
             m_secondaryCompositedLayerCount, m_obligatoryBackingStoreBytes / 1024, m_secondaryBackingStoreBytes / 1024, (m_obligatoryBackingStoreBytes + m_secondaryBackingStoreBytes) / 1024, (endTime - startTime).milliseconds());
+
+#if ENABLE(RESOURCE_USAGE)
+        // We save the total memory size of layers to show in the memory timeline of Inspector.
+        WebCore::ResourceUsageThread::setTotalLayerBackingStoreBytes(m_obligatoryBackingStoreBytes + m_secondaryBackingStoreBytes);
+#endif
```

Comment:
The problem with this is that it's only executed if LOG is enabled, which is odd. So in order to get a valid value we need to enable both LOG and RESOURCE_USAGE. We should be able to get this value without having to enable LOG.